### ISSUE TITLE
Multiple Monitor Support

### ISSUE THREAD

**Issue by lionkm** [linuxmint/mdm#95]
This is not a new issue, it's pointing out something the Mint community seems to be ignoring. Petra has 45 desktop effects to choose from, but can't do multiple monitors properly. Sure they can be configured with mdm...but only after login. It's bad enough that the login dialogue is replicated on multiple monitors, but monitors rotated 90 degrees present eye candy only a programmer could accept. Why did Mint choose to use mdm? At least with kdm, the problem can be corrected with xrandr commands in Xsetup. Different for the sake of being different is no better than change for the sake of change. Over 10 different versions of something as basic as a desktop manager, none of which can be configured without hacks/tweaks, will not get Linux converts.
